server/resources/[VORP]/[homestead_essentials]/homestead_jobs/server/functions.lua

```diff
@@ -0,0 +1,21 @@
+
+--- @param source integer
+--- @param job string
+--- @param jobgrade integer
+function validateJob(job, jobgrade)
+    local jobsData = LoadResourceFile(GetCurrentResourceName(), "shared/jobs.json")
+    jobsData = json.decode(jobsData)
+    local exists = false
+    for job, grades in pairs(jobsData.Jobs) do
+        if job == job then
+            for grade, data in pairs(grades) do
+                if grade == jobgrade then
```

Comment:
Is there a way to convert a list of entities into a table? Feels like a table would be more optimal than O(N^2)